engine/src/main/resources/assets/i18n/menu.lang

```diff
@@ -50,6 +50,8 @@
     "category-mouse": "category-mouse",
     "category-movement": "category-movement",
     "category-nui": "category-nui",
+    "change-popup-message": "change-popup-message",
```

Comment:
I'd go for a more descriptive key `change-keybind-popup-message` as "change" is quite general and could conflict with other change actions in the menu later. 
Another option would be move it to another .lang file specific for keybindings....